src/subdatatable/subdatatable.jl

```diff
@@ -157,4 +157,6 @@ end
 ##
 ##############################################################################
 
+Base.map(f::Function, sdt::SubDataTable) = f(sdt) # TODO: deprecate
```

Comment:
Same deal as before--if it's not needed we might as well just dump it rather than deprecate it. But why add it if you ultimately want to deprecate it anyway?